Eastern-Research-Group/mywaterway
# Feature/package upgrades breaking changes

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3376896

## Main Changes:
* Upgraded highcharts and fixed an issue with the survey results crashing the app (caused by upgrading highcharts).
* Upgraded react-table and converted all of the react-table components to use the new react-table api.

## Steps To Test:
1. Run `npm ci` from the client folder
2. Navigate to http://localhost:3000/state/KS/water-quality-overview
3. In the "Water Type" drop down, select "Rivers and Streams" 
4. Make sure the pie chart is visible and the app didn't crash. 
5. Expand the documents section and make sure the tables work correctly. 
    * Verify the tables look the same as the original tables. 
      Note: There may be some slight padding differences, which is ok. 
    * Verify the sorting works.
    * Verify the resizing works. 
      Note: The new react-table resizing works a little differently. The old version would shrink the last column before adding a horizontal scroll bar for the table. The new version just adds the horizontal scroll bar for the table.
6. Navigate to http://localhost:3000/attains
7. Make sure the table works correctly.
    * Verify the tables look the same as the original tables. 
      Note: There may be some slight padding differences, which is ok. 
    * Verify the sorting works.
    * Verify the resizing works. 
      Note: The new react-table resizing works a little differently. The old version would shrink the last column before adding a horizontal scroll bar for the table. The new version just adds the horizontal scroll bar for the table.
    * Verify the filtering works.
